src/api/app/models/workflow/step.rb

```diff
@@ -111,6 +111,10 @@ def create_subscriptions(package, workflow_filters)
     end
   end
 
+  def destroy_subscriptions(package)
```

Comment:
We should still rename the method so it's clear that it's calling `delete_all` and not `destroy_all`.

```suggestion
  def delete_all_subscriptions(package)
```

Reply:
I prefer without `_all_` thank you for the suggestion.